cryptofeed/exchanges/binance.py

```diff
@@ -66,6 +68,13 @@ def _parse_symbol_data(cls, data: dict) -> Tuple[Dict, Dict]:
             info['instrument_type'][s.normalized] = stype
         return ret, info
 
+    def _max_msgs_since_snapshot(self) -> int:
```

Comment:
This function decides when we should re-fetch the snapshot.

Currently it is based on the number of messages received. I'm not actually sure if that's a good idea since number of received messages != book updates. Perhaps we'd be better off on basing this on a timer instead?